src/components/search/search.component.tsx

```diff
@@ -229,6 +235,7 @@ export const Search = ({
     <StyledSearch
       isFocused={isFocused}
       searchWidth={searchWidth}
+      maxWidth={maxWidth}
```

Comment:
could we not pass the `maxWidth` prop to the child `Textbox` rather than to the `StyledSearch`? This would mean we don't have to make any changes to `StyledSearch` (and in particular we wouldn't have to copy changes that are already in `InputPresentation`).

Reply:
I _think_ this will make the layout look a bit weird and Tom's approach here is correct

Reply:
This has been left in as the width of the full container needed to be changed, so the search button is also included, a similar approach has also been taken for the `searchWidth` prop. Just passing the `maxWidth` prop to `Textbox` would change the `max-width` of the input but the search button would not stay grouped with the input.